### PR TITLE
Remove menudata backup

### PR DESCRIPTION
**Pull request by FazanaJ**
Not considered important enough to keep

### CODE REVIEW

File: include/config/config_rom.h [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 // Internal ROM name. NEEDS TO BE **EXACTLY** 20 CHARACTERS. Can't be 19 characters, can't be 21 characters. You can fill it with spaces.
 // The end quote should be here:               "
-#define INTERNAL_ROM_NAME "SUPER MARIO 64      "
+#define INTERNAL_ROM_NAME "HackerSM64          "
```
Comment (thecozies) [inline include/config/config_rom.h:9]: I'm cool with this, anyone else's thoughts?

Comment (FazanaJ) [reply include/config/config_rom.h:9]: Oh right, for an explanation, iirc VC will break if we don't rename the rom lol

**Comment by Mr-Wiseguy** [comment]
What has this been tested on?

**Review by thecozies — APPROVED** [review]
should be fine

File: include/config/config_rom.h (hunk 1 above)
Comment (Reonu) [reply include/config/config_rom.h:9]: Fine by me too

**Review by Reonu — APPROVED** [review]
I can't test console myself rn but it has been tested already

Emu works

lgtm

**Review by Mr-Wiseguy — APPROVED** [review]
Lgtm

**Pull request merged**